cmd/templates_test.go

```diff
@@ -33,7 +33,9 @@ rust         http
 springboot   cloudevents
 springboot   http
 typescript   cloudevents
-typescript   http`
+typescript   http
+csharp		 cloudevents
```

Comment:
Cloudevents are mentioned couple of time but it seems this adds only http template.

Reply:
@matejvasek  

> Instead I just went and added new C# runtime that currently only supports http but you are welcome to add cloudevents.

Yes I didn't dive much into cloudevents but they can be taken from [here](https://github.com/knative/docs/tree/main/code-samples/serving/cloudevents/cloudevents-dotnet)